config/settings/base.py

```diff
@@ -161,38 +161,6 @@
 
 GTM_CONTAINER_ID = get_env_variable("GTM_CONTAINER_ID", "")
 
-##########
-# Mapbox #
-##########
-
-# https://www.mapbox.com/studio/account/tokens/
-
-MAPBOX_ACCESS_TOKEN = get_env_variable("MAPBOX_ACCESS_TOKEN", "")
-
```

Comment:
Weren't we going to eventually keep Mapbox and let the athlete select? Maybe we can keep the token until then?

Reply:
I'm not a big fan of "keep dead code in case of..". It's easy enough to recover code with git. Unless you want to add support for multiple layers in that issue?

Reply:
True that! I thought maybe you would like to try it.